<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GammaRay::RecordingWidget</class>
 <widget class="QWidget" name="GammaRay::RecordingWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>947</width>
    <height>535</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="toolbarLayout">
     <property name="bottomMargin">
      <number>6</number>
     </property>
     <item>
      <widget class="QLineEdit" name="searchLine">
       <property name="placeholderText">
        <string>Filter...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="recordAllButton">
       <property name="text">
        <string>Record All</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="recordNoneButton">
       <property name="text">
        <string>Record None</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Maximum</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>50</width>
         <height>0</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QToolButton" name="showAllButton">
       <property name="text">
        <string>Show All</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="showNonoeButton">
       <property name="text">
        <string>Show None</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="GammaRay::DeferredTreeView" name="treeView">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>10</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GammaRay::DeferredTreeView</class>
   <extends>QTreeView</extends>
   <header location="global">ui/deferredtreeview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
